App/Components/Interaction/CompAddFriend.razor

```diff
@@ -0,0 +1,139 @@
+@inherits UserViewComponentBase
+@implements IInteractionSubscriber
+
+@using Bamboozlers.Classes.AppDbContext
+@using Bamboozlers.Classes.Data
+@using Bamboozlers.Classes.Services
+@using Bamboozlers.Classes.Services.UserServices
+@using Bamboozlers.Classes.Utility.Observer
+@using Bamboozlers.Components.Utility
+@using Blazorise.Extensions
+
+<ModalContent Centered Size="ModalSize.Large">
+    <ModalHeader>
+        <ModalTitle>Add Friends</ModalTitle>
+    </ModalHeader>
+    <ModalBody>
+        <Heading>Find users by their username or display name.</Heading>
+        <Divider/>
+        <Div Style="display: block">
+            <div style="display: block;">
+                <TextEdit @bind-Text="SearchQuery"
```

Comment:
Pressing enter should search, my branch has a keypress service listener thingy if you want to defer that to post merge